### PR TITLE
buildqtc.sh: Bundle pkg-config on Windows

### PR DESCRIPTION
**Pull request by martyone**

### CODE REVIEW

File: buildqtc.sh [hunk 1]
```diff
@@ -408,6 +408,13 @@ build_windows_qtc() {
 	local opengl32sw_lib="opengl32sw-32.7z"
 	curl -s -f -o $opengl32sw_lib http://$OPT_UPLOAD_HOST/sailfishos/win32-binary-artifacts/opengl/$opengl32sw_lib
 
+    local pkg_config_libs="pkg-config_0.26-1_win32.zip \
+        gettext-runtime_0.18.1.1-2_win32.zip \
+        glib_2.28.8-1_win32.zip"
```
Comment (vigejolla) [inline buildqtc.sh]: Where do these come from? Is it documented somewhere?

Comment (martyone) [reply buildqtc.sh]: will be documented in sdk-docs

Comment (martyone) [reply buildqtc.sh]: but we have the README as well - I will update it in a sec

**Comment by martyone** [comment]
> For some reason I didn't get any notification from github that you had pushed changes...

It never does so

**Comment by vigejolla** [comment]
> > For some reason I didn't get any notification from github that you had pushed changes...
> 
> It never does so

Funny, I'm quite sure I have received notifications of such events before. I must have been imagining things...

**Comment by martyone** [comment]
> Funny, I'm quite sure I have received notifications of such events before. I must have been imagining things...

Same here :)

**Pull request merged**